packages/overture-schema/README.md

```diff
@@ -1,106 +1,93 @@
 # overture-schema
 
-Overture Maps schema collection with implemented themes.
+Type-safe Python models for [Overture Maps Foundation](https://overturemaps.org/) data.
 
-This package provides testing infrastructure for Overture Maps Pydantic schemas. It includes a pytest-based test harness that validates schema models against curated examples and counterexamples.
+This package provides Pydantic models for validating and working with Overture Maps data, including buildings, places, addresses, transportation networks, and administrative boundaries.
```

Comment:
Add reference to base?

Reply:
I'm struggling to characterize the data that lives in base in a useful way to include here. I figure we'll probably touch this a few times, and "including" does a bunch of work.